File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int s = n - 1; s > i; s--)
        {
             printf(" ");
        }
        for (int l = -1; l < i; l++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = -1; l < i; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

n = 0
while n < 1 or n > 8:
    n = get_int("Height: ")
for i in range(0, n, 1):

    for s in range(n - 1, i, -1):
        print(" ", end="")

    for l in range(-1, i, 1):
        print("#", end="")

    print("  ", end="")

    for l in range(-1, i, 1):
        print("#", end="")

    print()